message_parser/src/main/java/cumulus_message_adapter/message_parser/JsonUtilities.java

```diff
@@ -0,0 +1,37 @@
+package cumulus_message_adapter.message_parser;
+
+import java.util.Map;
+
+import java.lang.reflect.Type;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * Utilities for converting JSON to string/map
+ */
+public class JsonUtilities {
```

Comment:
Minor rename suggestion: `Json` or `JsonUtils`. A `Utils` suffix is common in Java, but not `Utilities` spelled out. In this case, I actually recommend simply `Json` because I believe things will read better with my suggested method renames (see other comments).